README.md

```diff
@@ -64,6 +64,27 @@ These are the final system requirements: `Cairo`, `Latex`, `ffmpeg` and `sox`.
 
 ### Windows Users
 
+!!!to my knowledge manim only works with python 3.7 (yet)!!!
```

Comment:
is this correct or is my knowledge outdated?

Reply:
I believe it is possible to do with 3.8 but not super easily as you will have  to `pip install` a bunch of different packages.

Reply:
Oh, okay, so well just write that it is only for 3.7 or something and the guide is not guranteed to work in later versions

Reply:
👍 Right, and then later, it can be updated. Your tutorial looks accurate as well.